### DISCUSSION TITLE
fwidth of Gaussian source

### DISCUSSION THREAD

**Discussion by mingoosam** [NanoComp/meep#2733] [Q&A]
Hello! When I define a Gaussian source with `fcen` and an `fwidth` parameters, does meep build the Gaussian with `fwidth` as the standard deviation or the full width half max?

**Comment by zekishaw** [comment]
https://meep.readthedocs.io/en/latest/Python_User_Interface/#gaussiansource

I assume you have looked already but there is info here in the docs on the Gaussian source

**Reply by smartalecH** [reply]
It may not be clear from the docs, but the source of truth is always the code:

https://github.com/NanoComp/meep/blob/e5f88e5d6db24e5b49ea1de68fc45267a7452b56/src/sources.cpp#L106

So `width` is the standard deviation of the gaussian.

**Reply by stevengj** [reply]
Note that `width = 1/fwidth`
